File: exercise/kotlin/week02-exercise2/src/main/kotlin/Book.kt
class Book(var title: String, var author: String, var copies: Int) {

    fun addCopies(additionalCopies: Int) {
        if (additionalCopies > 0) {
            this.copies += additionalCopies
        }
    }

    fun removeCopies(soldCopies: Int) {
        if (soldCopies > 0 && this.copies >= soldCopies) {
            this.copies -= soldCopies
        }
    }
}

File: exercise/c#/week02-object-calisthenics/BookStore/Book.cs
namespace BookStore
{
    public class Book
    {
        public string Title { get; set; }
        public string Author { get; set; }
        public int Copies { get; set; }

        public Book(string title, string author, int copies)
        {
            Title = title;
            Author = author;
            Copies = copies;
        }

        public void AddCopies(int additionalCopies)
        {
            if (additionalCopies > 0)
            {
                Copies += additionalCopies;
            }
        }

        public void RemoveCopies(int soldCopies)
        {
            if (soldCopies > 0 && Copies >= soldCopies)
            {
                Copies -= soldCopies;
            }
        }
    }
}